requirements.txt

```diff
@@ -1,12 +1,17 @@
+click==8.0.3
 Flask==1.1.2
+Flask-Images==3.0.2
 Flask-Login==0.5.0
 Flask-SQLAlchemy==2.4.4
 gunicorn==20.0.4
+honcho==1.0.1
 itsdangerous==1.1.0
 Jinja2==2.11.2
 MarkupSafe==1.1.1
-#psycopg2==2.6.1
+PIL==1.1.7
+Pillow==8.4.0
+pillowcase==2.0.0
+pipreq==0.4
+six==1.16.0
 SQLAlchemy==1.3.20
 Werkzeug==1.0.1
-wheel==0.36.0
```

Comment:
wheel may be an issue?

Reply:
Wheel is actually superfluous for the requirements.txt it seems, so this is clean.